ci-operator/step-registry/layered-product/odf/aws/add-storage/layered-product-odf-aws-add-storage-commands.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+set -x
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+# create the policies namespace
+oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-storage
+EOF
+
+oc label namespace openshift-storage openshift.io/cluster-monitoring=true
+
+
+# create 6 machinesets for ocp storage on aws
+
+CLUSTERID=$(oc get machineset -n openshift-machine-api -o jsonpath='{.items[0].metadata.labels.machine\.openshift\.io/cluster-api-cluster}')
+AZ=$(oc get machineset -n openshift-machine-api -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.placement.availabilityZone}')
+REGION=$(oc get machineset -n openshift-machine-api -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.placement.region}')
+VOLUME_TYPE=$(oc get machineset -n  openshift-machine-api -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.blockDevices[0].ebs.volumeType}')
+INSTANCE_TYPE=$( oc get machineset -n  openshift-machine-api -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.instanceType}')
+echo $AZ
+echo $REGION
+echo $CLUSTERID
+echo $VOLUME_TYPE
+echo $INSTANCE_TYPE
+
+
+oc apply -f - <<EOF 
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: $CLUSTERID
+    machine.openshift.io/cluster-api-machine-role: workerocs
+    machine.openshift.io/cluster-api-machine-type: workerocs
+  name: $CLUSTERID-workerocs-$AZ
+  namespace: openshift-machine-api
+spec:
+  replicas: 7
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: $CLUSTERID
+      machine.openshift.io/cluster-api-machineset: $CLUSTERID-workerocs-$AZ
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        machine.openshift.io/cluster-api-cluster: $CLUSTERID
+        machine.openshift.io/cluster-api-machine-role: workerocs
+        machine.openshift.io/cluster-api-machine-type: workerocs
+        machine.openshift.io/cluster-api-machineset: $CLUSTERID-workerocs-$AZ
+    spec:
+      metadata:
+        creationTimestamp: null
+        labels:
+          cluster.ocs.openshift.io/openshift-storage: ""
+          node-role.kubernetes.io/worker: ""
+      providerSpec:
+        value:
+          ami:
+            id: ami-0cd5c6a0f8bb3c33d
```

Comment:
Do you know if this ami id matches the $INSTANCE_TYPE that you are using?

Reply:
I don't think I guess it doesn't affect